scene_synchronizer.cpp

```diff
@@ -2440,29 +2440,40 @@ void ServerSynchronizer::execute_actions() {
 		const int sender_peer = server_actions[i].sender_peer;
 		uint32_t executed_input_id = UINT32_MAX;
 		if (sender_peer == 1) {
-			// Triggered by the server so any `controller` will work just fine: Takes the first one.
 			ERR_CONTINUE_MSG(scene_synchronizer->peer_data.iter().valid == false, "[FATAL] It's not supposed to have no peers.");
```

Comment:
Quick note about this condition here, what happens if all of the peers have disconnected? Shouldn't the no peer case be handled?

Reply:
It should be impossible to trigger because when the peer is removed the actions are cleared too. If no peers, the action is not generated. Tomorrow I’ll double check and eventually fix it if it’s not the case